src/Tribe/Attendee_Registration/View.php

```diff
@@ -37,170 +25,14 @@ public function display_attendee_registration_page() {
-
-			if ( empty( $default_provider ) ) {
-				// One provider per instance.
-				$default_provider[ $q_provider ] = $ticket->provider->class_name;
-			}
-
-			/** @var Tribe__Tickets__Status__Manager $status */
-			$status   = tribe( 'tickets.status' );
-			$provider = $status->get_provider_slug( $ticket->provider->class_name );
-
-			$providers[ $ticket->event ] = $provider;
-			$events[ $ticket->event ][]  = $ticket_data;
-		}
-
-		/**
-		 * Check if the cart has a ticket with required meta fields
-		 *
-		 * @since TDB
-		 *
-		 * @param boolean $cart_has_required_meta Whether the cart has required meta.
-		 * @param array   $tickets_in_cart        The array containing the cart elements. Format array( 'ticket_id' => 'quantity' ).
-		 */
-		$cart_has_required_meta = (bool) apply_filters( 'tribe_tickets_attendee_registration_has_required_meta', ! empty( $tickets_in_cart ), $tickets_in_cart );
-
-		/** @var Tribe__Tickets__Attendee_Registration__Main $attendee_registration */
-		$attendee_registration = tribe( 'tickets.attendee_registration' );
-
-		// Get the checkout URL, it'll be added to the checkout button.
-		$checkout_url = $attendee_registration->get_checkout_url();
-
-		/**
-		 * Filter to check if there's any required meta that wasn't filled in
-		 *
-		 * @since TDB
-		 *
-		 * @param bool
-		 */
-		$is_meta_up_to_date = (int) apply_filters( 'tribe_tickets_attendee_registration_is_meta_up_to_date', true );
-
-		// Enqueue styles and scripts for this page.
-		tribe_asset_enqueue_group( 'tribe-tickets-registration-page' );
-
-		// One provider per instance.
-		$currency        = tribe( 'tickets.commerce.currency' );
-		$currency_config = $currency->get_currency_config_for_provider( $default_provider, null );
-
-		/**
-		 *  Set all the template variables
-		 */
-		$args = [
-			'events'                 => $events,
-			'checkout_url'           => $checkout_url,
-			'is_meta_up_to_date'     => $is_meta_up_to_date,
-			'cart_has_required_meta' => $cart_has_required_meta,
-			'providers'              => $providers,
-			'currency'               => $currency,
-			'currency_config'        => $currency_config,
-			'is_modal'               => null,
-			'non_meta_count'         => $non_meta_count,
-		];
-
-		if ( tribe_tickets_new_views_is_enabled() ) {
-			$provider = $this->get( 'provider' ) ?: tribe_get_request_var( 'provider' );
-
-			if ( empty( $provider ) ) {
-				$event_keys   = array_keys( $events );
-				$event_key    = array_shift( $event_keys );
-				$provider_obj = Tribe__Tickets__Tickets::get_event_ticket_provider_object( $event_key );
-			} elseif ( is_string( $provider ) ) {
-				$provider_obj = $this->get_cart_provider( $provider );
-			} elseif ( $provider instanceof Tribe__Tickets__Tickets ) {
-				$provider_obj = $provider;
-			}
-
-			if ( $provider_obj instanceof Tribe__Tickets__Tickets ) {
-				$provider = $provider_obj->attendee_object;
-			}
+		try {
+			/* @var Tribe__Tickets_Plus__Attendee_Registration__View $view */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: b42c2b52c148fdcd2c99e123787cdc46</sub>